assembl/alembic/versions/c98a9b6f6b7f_add_sections.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding=utf-8 -*-
+"""add_sections
+
+Revision ID: c98a9b6f6b7f
+Revises: ca1c445a2e24
+Create Date: 2017-11-17 15:10:13.609004
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c98a9b6f6b7f'
+down_revision = 'ca1c445a2e24'
+
+from alembic import context, op
+import sqlalchemy as sa
+import transaction
+
+
+from assembl.lib import config
+from assembl.lib.sqla_types import URLString
+
+
+def upgrade(pyramid_env):
+    with context.begin_transaction():
+        pass
+
+    # Do stuff with the app's models here.
+    from assembl import models as m
+    from assembl.models.section import SectionTypesEnum
+
+    db = m.get_session_maker()()
+    with transaction.manager:
+        section_types = [
+            t.value for t in SectionTypesEnum.__members__.values()]
+        op.create_table(
+            'section',
+            sa.Column('id', sa.Integer, primary_key=True),
+            sa.Column('type', sa.String(60), nullable=False),
+            sa.Column('discussion_id',
+                      sa.Integer,
+                      sa.ForeignKey(
+                          'discussion.id',
+                          ondelete="CASCADE",
+                          onupdate="CASCADE"), nullable=False, index=False),
+            sa.Column('title_id', sa.Integer, sa.ForeignKey('langstring.id')),
+            sa.Column('url', URLString(1024)),
+            sa.Column('order', sa.Float, default=0.0),
```

Comment:
You should set nullable=False for order.

Reply:
done